### ISSUE TITLE
Export button and Uplink are not working. 

### ISSUE THREAD

**Issue by BouTayehGaby** [gotthardp/lorawan-server#620]
The uplink was working fine when two sensors were sending simultaneously, I was able to send data to my application server (Web application on a hosting server).

I deployed 15 sensors and the Network server stopped transmitting the received frames to my application server.

Therefore I tried to use the export button in Frames but it is not working!

current version : 0.6.3.

Thank you

**Comment by gotthardp** [comment]
Hello. I confirm the Export of rxframes is not working. I just fixed that.
To debug the "uplink not working" issue I'd need more information like debug logs etc.

**Comment by BouTayehGaby** [comment]
![Error Server 961](https://user-images.githubusercontent.com/24385183/55222589-971f3200-520c-11e9-899e-e969e8d665b1.jpg)

As seen in the image, on the recieving server side (where my application is located) my IP was black listed since the requests header contained illegal characters (Characterized as an intrusion attack on the server).  
It is weird since I was able to send data previously without any problem.

Any ideas what could have generated such an issue?

**Comment by gotthardp** [comment]
I cannot tell you more than what you can read in the log you posted. :-) I will close this issue now as the first problem was fixed and the second is not related to the lorawan server.

**Comment by BouTayehGaby** [comment]
isn't it however related to the content of the header in the transmitted packet generated by the lorawan server?

**Comment by gotthardp** [comment]
true, but do you know what did the server send?

**Comment by BouTayehGaby** [comment]
Which server are you referring to? the lorawan server?

**Comment by gotthardp** [comment]
yes. Could you please track what the lorawan-server did send that made the IDS to stop the traffic?
